src/mixer/basetrackplayer.h

```diff
@@ -100,6 +107,10 @@ class BaseTrackPlayerImpl : public BaseTrackPlayer {
     ControlObjectSlave* m_pPreGain;
     ControlObjectSlave* m_pSpeed;
     ControlObjectSlave* m_pPitchAdjust;
+    QScopedPointer<ControlObjectSlave> m_pInputEnabled;
+    QScopedPointer<ControlObjectSlave> m_pPassthroughEnabled;
+    QScopedPointer<ControlObjectSlave> m_pVinylControlEnabled;
+    QScopedPointer<ControlObjectSlave> m_pVinylControlStatus;
```

Comment:
It is not worth to discuss this in the context of i5/i7 and Mixxx, the differences are only tiny and a maintainable code is more important. But I am an original embedded developer, so I wanted to know what is correct. I compiled some test files and the result is interesting:  

In this context it is obvious, that a redundant delete is more asm code than without. 
It dependents on the implementation of the Qt object tree if it consumes more cpu cycles or not. 
I did not look at this.    

This QScopedPointer member variables are consuming much more cpu cycles, since we 
call the default constructor fist and then we call reset to adopt the final pointer. I was also surprised to see "call" statements in the asm output. This is due to the PIMPL pattern used for  QScopedPointer, instead of using an inline implementation as I have original assumed 

An interesting remaining question how compares a new/delete and QScopedPointer and std::unique_ptr in function scope. std::unique_ptr is inline implemented and the compiler is able to optimize it a lot, but it still takes a hand full  more cycles then a new/delete solution because of its custom deleter feature. Only a simple self implemented Scoped pointer produces exactly the same asm code that the original new/delete version. 

Conclusion:  
prefere std::unique_ptr over QScopedPointer